packages/iov-bcp/src/atomicswaptypes.ts

```diff
@@ -22,9 +21,6 @@ export function isSwapProcessStateAborted(state: SwapProcessState): state is Swa
   return state === SwapProcessState.Aborted;
 }
 
-export type Preimage = Uint8Array & As<"preimage">;
-export type Hash = Uint8Array & As<"hash">;
```

Comment:
Those are needed for `SwapOfferTransaction`/`SwapClaimTransaction`. Doesn't it make more sense to move the atomic swap transaction types into `atomicswaptypes.ts` instead? This should be very easy now that transaction type registraion is decentralized. This was not always the case: `transactions.ts` has some history in which UnsignedTransaction was a union type.